server/src/packages/pas-form/pas-form.service.ts

```diff
@@ -23,7 +23,7 @@ export class PasFormService {
 
     // dcp_projectname is the name of the CRM navigation link.
     // then dcp_projectname is the real project name
-    const { dcp_projectname } = pasForm.dcp_projectname;
+    const { dcp_projectname } = pasForm.dcp_projectname || {};
```

Comment:
The commit description says this is to fix something about CRM's return values for related entities. I'm not following.. how does this relate to dcp_projectname?

Reply:
It doesn't, I just accidentally branched off of that. That's addressed in another PR.

Reply:
This PR is about validating hasMany length. The diff here is for another PR: https://github.com/NYCPlanning/labs-applicant-portal/pull/529

Reply:
👍  Gotcha